pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -57,7 +57,7 @@
 public abstract class ControllerTest {
   public static final String LOCAL_HOST = "localhost";
 
-  private static final int DEFAULT_CONTROLLER_PORT = 8998;
+  private static final int DEFAULT_CONTROLLER_PORT = 9550;
```

Comment:
Can you comment on why changing the port in the commit? (Encountering issue running tests in IDE?)

Reply:
Some background process is running and occupying the port number. It's been addressed.